### DISCUSSION TITLE
Could not load file or assembly. ¿Error?

### DISCUSSION THREAD

**Discussion by DilbertRV** [Taiizor/ReaLTaiizor#39] [Q&A] [answered]
Hi!
How can i resolve this problem? 
No controls appear in my toolbox and i got this message. 

![image](https://user-images.githubusercontent.com/63085942/130344221-cf28e464-25de-4ad2-a52f-99783e3354b4.png)

Can someone help me? Thanks!

**Comment by Taiizor** [comment, marked as answer]
Hi,
Did you add ReaLTaiizor to your project via NuGet?

**Reply by DilbertRV** [reply]
Thanks for responding, but yes, I did.
The strange thing is that when cloning the code on a new PC, the controls were not showing up in the toolkit.
I think part of the problem was the path assembly, which I still don't know much about, but the .dll's were not referenced even through via NuGet.

- The solution was to change the project from netcore 3.1 to a Framework 4.8 version, with that, the controls appeared in the toolkit.  A bit risky for what I already had from the project, but it worked for me. 👍

**Reply by Taiizor** [reply]
In .NET Framework projects, NuGet references are kept in the project's folder location.
But in .NET (Core) applications, NuGet references are kept in a different directory.
Therefore, you may encounter such errors when you open the project on a different computer.

I ran the library in .NET Core 3.1 application without any problem as you can see below.
https://imgur.com/787cWV6
(https://user-images.githubusercontent.com/41683699/130370532-ffab479e-46b3-404d-9082-1cd25d93104c.mp4)
